src/main/scala/scalismo/utils/Conversions.scala

```diff
@@ -153,6 +155,100 @@ object VtkHelpers {
 
 }
 
+object TetrahedronMeshConversion {
+
+  private def extractPointsAndCells(ug: vtkUnstructuredGrid, calculateNewDelaunayTetrahedrons: Boolean = false) = Try {
```

Comment:
I find the calculateNewDelauneyTetrahedrons here out of place. We are in the conversion method, which converts structures and should not change them. I would find it better to have this as part of the mesh operations.

Reply:
I agree that this should be moved or even dropped, at least for the moment. As it is not IO I will drop it from the PR for the moment.